src/lib/previews.ts

```
@@ -0,0 +1,133 @@
+  width: number;
+  height: number;
+}
+export interface pdfToImgProps {
+  blob: string;
+  width: number;
+  height: number;
+}
+
+export async function getPDFPreview(
+  file: File,
+  firstPageOnly: Boolean = true,
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)